editor/src/clj/editor/cljfx_form_view.clj

```diff
@@ -465,7 +465,7 @@
      :on-value-changed on-value-changed
      :converter (DefoldStringConverter.
                   #(get value->label % (to-string %))
-                  #(get label->value % (and from-string (from-string %))))
+                  #(or (label->value %) (and from-string (from-string %))))
```

Comment:
The previous code was incorrect, but the error was never triggered before because option types were always strings. Now there is an integer option type (`graphics.opengl_version_hint`) with some predefined options like `OpenGL 3.3`. The problem: string like `OpenGL 3.3` should only be looked up in `label->value`, and should not be converted to int from a string, which was always happening before.